packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -202,6 +208,24 @@ describe('logs', () => {
 
       resetExperimentalFeatures()
     })
+
+    it('should send reports', () => {
+      if (!isChromium()) {
+        pending('no ReportingObserver support')
+      }
+      const logger = new Logger(noop)
+      const logErrorSpy = spyOn(logger, 'log')
+      const reportingObserverStub = stubReportingObserver()
+      updateExperimentalFeatures(['forward-reports'])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a test without the FF to ensure that it also behave correctly?

Reply:
Sure